app/code/Magento/Sales/Block/Order/Totals.php

```diff
@@ -198,7 +198,10 @@ public function addTotal(\Magento\Framework\DataObject $total, $after = null)
             }
             if (!$added) {
                 $last = array_pop($totals);
-                $totals[$total->getCode()] = $total;
+                // Push tax before grand total.
+                $totals =  array_slice($totals, 0,2, true) +
```

Comment:
please, insert a space after `0,` to comply with coding styles, thanks!

Reply:
Done, in my IDE i didn't see that :)
![DeepinScreenshot_select-area_20190318105921](https://user-images.githubusercontent.com/2481206/54518452-f8bde180-496c-11e9-91ae-94ef3e18f84d.png)